.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v**"
```

Comment:
also includes typos like `v.2.3.1` and other tags with 'v' like `vroom` - but not really harmful.